explorer/app/components/Table/table.tsx

```diff
@@ -46,23 +56,58 @@ export const Table = <T extends object>({
     {
       columns,
       data: items,
+      disableMultiSort: true,
       initialState: {
         pageSize: pageSize,
       } as TableState,
       manualPagination: !!pagination,
+      manualSortBy: true,
       pageCount: total,
     },
+    useSortBy,
     usePagination
   );
 
+  const handleSortClicked = (column: ColumnInstance<T>) => {
+    const newColumn =
```

Comment:
Can we pull out `newColumn` and `newOrder` into functions so we can test them independently? Same with the `direction` prop of `TableSortLabel` below.